pkgs/applications/networking/cisco-packet-tracer/7.nix

```diff
@@ -0,0 +1,91 @@
+{ stdenv
+, lib
+, dpkg
+, lndir
+, makeWrapper
+, buildFHSUserEnvBubblewrap
+, makeDesktopItem
+, callPackage
+, requireFile
+}:
+
+let
+  version = "7.3.1";
+
+  ptFiles = stdenv.mkDerivation {
+    name = "PacketTracer7drv";
+    inherit version;
+
+    dontUnpack = true;
+    src = requireFile {
+      name = "PacketTracer_${builtins.replaceStrings ["."] [""] version}_amd64.deb";
+      sha256 = "c39802d15dd61d00ba27fb8c116da45fd8562ab4b49996555ad66b88deace27f";
+      url = "https://www.netacad.com";
+    };
+
+    nativeBuildInputs = [ dpkg makeWrapper ];
+
+    installPhase = ''
+      dpkg-deb -x $src $out
+      makeWrapper "$out/opt/pt/bin/PacketTracer7" "$out/bin/packettracer7" \
+          --prefix LD_LIBRARY_PATH : "$out/opt/pt/bin"
+    '';
+  };
+
+  desktopItem = makeDesktopItem {
```

Comment:
Please use copyDesktopItems.

Reply:
This is by purpose, doing it otherwise make it unable to have both versions installed in parallel

Both desktop files reference a packettracer binary, I've used packettracer{7,8} to solve this question.